doctor/lib/mojibake.py

```diff
@@ -47,7 +47,6 @@ def fix_mojibake(text):
         "Ò": "N",
         "Ñ": "O",
         "Ð": "P",
-        "": "Q",  # Missing
```

Comment:
Key is overwritten in L56

Reply:
That's actually deliberate to document how we don't have the replacement keys for Q or X.

Reply:
@mlissner fixed now. Should be good to go.